src/MapfileStyleParser.ts

```diff
@@ -824,6 +833,29 @@ export class MapfileStyleParser implements StyleParser {
     return symbolizers;
   }
 
+  /**
+   * Collect icon data if defined within the label tag
+   *
+   * @param mapfileLabel
+   * @returns
+   */
+  getIconsFromMapfileLabel(
+    mapfileLabel: MapfileLabel
+  ): IconSymbolizer[] | void {
+    const labelStyles: IconSymbolizer[] = [];
+    mapfileLabel?.styles?.forEach((style: MapfileStyle) => {
+      if (style.symbol) {
+        labelStyles.push(this.getIconSymbolizerFromMapfileStyle(style));
+      }
+    });
+
+    if (labelStyles.length) {
```

Comment:
I would prefer to just return an empty array and do the check in the calling method

Reply:
I have adjusted the return process.